### PR TITLE
Accepted 1 and Opened 2 PRs

### PR DESCRIPTION
**Pull request by Derrick2000**

### CODE REVIEW

File: pr-data.csv [hunk 1]
```diff
@@ -2686,7 +2686,7 @@ https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1a
...
 https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-model/drools-model-codegen,org.drools.model.codegen.execmodel.GroupByTest.testDecomposedGroupByKey,ID,,,
 https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-model/drools-model-codegen,org.drools.model.codegen.execmodel.GroupByTest.testDecomposedGroupByKeyAnd2Accumulates,ID,,,
-https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-model/drools-model-codegen,org.drools.model.codegen.execmodel.GroupByTest.testDecomposedGroupByKeyAnd2AccumulatesInConsequence,ID,,,
+https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-model/drools-model-codegen,org.drools.model.codegen.execmodel.GroupByTest.testDecomposedGroupByKeyAnd2AccumulatesInConsequence,ID,Opened,https://github.com/apache/incubator-kie-drools/pull/6119,The fix is identical to one of the opened PR and I choose not to create a new PR to avoid duplication.
```
Comment (darko-marinov) [inline pr-data.csv:2689]: What evidence (e.g., log files) do you have that the fix for another test also fixes this test?

File: pr-data.csv [hunk 2]
```diff
@@ -3652,7 +3652,7 @@ https://github.com/danfickle/openhtmltopdf,4a0612fab7b3a55e3e11d485233b063006d03
 https://github.com/DataDog/datadog-kafka-connect-logs,e453f0c9efb50f2d7aaf2186ff024b15bca9742b,.,com.datadoghq.connect.logs.sink.DatadogLogsApiWriterTest.writer_readingMultipleTopics_shouldBatchSeparate,ID,Opened,https://github.com/DataDog/datadog-kafka-connect-logs/pull/28,
 https://github.com/DataDog/jmeter-datadog-backend-listener,4b237366bf81050e56aa4d96944cffcd594593c0,.,org.datadog.jmeter.plugins.DatadogBackendClientTest.testExtractLogs,ID,Accepted,https://github.com/DataDog/jmeter-datadog-backend-listener/pull/33,
 https://github.com/DataDog/jmxfetch,15e72e2342dc29d71243308cdad8c525b4d34317,.,org.datadog.jmxfetch.TestApp.testApp,ID,,,
-https://github.com/DataDog/jmxfetch,15e72e2342dc29d71243308cdad8c525b4d34317,.,org.datadog.jmxfetch.TestApp.testAppCanonicalRate,ID,,,
+https://github.com/DataDog/jmxfetch,15e72e2342dc29d71243308cdad8c525b4d34317,.,org.datadog.jmxfetch.TestApp.testAppCanonicalRate,ID,Opened,https://github.com/DataDog/jmxfetch/pull/541,
```
Comment (darko-marinov) [inline pr-data.csv:3655]: I can accept this easily, but you may want to squash commits in that PR.

File: pr-data.csv [hunk 3]
```diff
@@ -4747,7 +4747,7 @@ https://github.com/JodaOrg/joda-time,d1ea2a53929d7d56d4f4560852e5586517a0dd47,.,
 https://github.com/jolokia/jolokia,ead322cc39f43b5b46d9353f539896a430a61b69,jolokia-jvm,org.jolokia.jvmagent.JolokiaServerTest.http,ID,,,
 https://github.com/jolokia/jolokia,ead322cc39f43b5b46d9353f539896a430a61b69,jolokia-jvm,org.jolokia.jvmagent.JolokiaServerTest.serverPicksThePort,ID,,,
 https://github.com/jolokia/jolokia,ead322cc39f43b5b46d9353f539896a430a61b69,jolokia-jvm,org.jolokia.jvmagent.JolokiaServerTest.sslWithSpecialHttpsSettings,ID,,,
-https://github.com/jolokia/jolokia,ead322cc39f43b5b46d9353f539896a430a61b69,jolokia-spring,org.jolokia.jvmagent.spring.JolokiaServerIntegrationTest.simple,ID,,,
+https://github.com/jolokia/jolokia,ead322cc39f43b5b46d9353f539896a430a61b69,jolokia-spring,org.jolokia.jvmagent.spring.JolokiaServerIntegrationTest.simple,ID,Accepted,https://github.com/jolokia/jolokia/pull/750,Merged
```
Comment (darko-marinov) [inline pr-data.csv:4750]: Please remove `Merged` from the note field (last).

**Review by darko-marinov — COMMENTED** [review]
You may want to split bigger PRs into smaller than can be easier accepted.

File: pr-data.csv (hunk 1 above)
Comment (Derrick2000) [reply pr-data.csv]: Will resolve in a separated PR

File: pr-data.csv (hunk 3 above)
Comment (Derrick2000) [reply pr-data.csv:4750]: Done

File: pr-data.csv (hunk 2 above)
Comment (Derrick2000) [reply pr-data.csv]: Will resolve in a separated PR

**Pull request merged**